### DISCUSSION TITLE
v2.0.31

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#327] [Announcements]
## Changes

- Fix missing devices @DTTerastar (#325)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.31'>v2.0.31</a>.</em>

**Comment by crhymes** [comment]
Just some positive feedback - everything working over here! 2x Mi Band 6 identified and tracking correctly.

**Reply by DTTerastar** [reply]
Awesome!  Everyone close your issues if it's fixed!

**Comment by tasi333** [comment]
So far so Stable, but multiple fitbit devices (and I hear also garmin devices) show up with the same signature.  Would be nice to add the mac address to the signature ID to help differentiate the devices.

Thanks for all your hard work @DTTerastar

**Reply by DTTerastar** [reply]
I think adding a static macs option would be good, if it matches that it doesn't fingerprint since i don't think 100% of fitbits have static mac addresses.
